docs/content/configuration.md

```diff
@@ -139,8 +139,10 @@ Other types of route groups and detailed information are available [in the speci
 
 By default, all traffic is denied so we need to grant access to clients to our application. This is done by defining a `TrafficTarget`.
 
-??? Note "TrafficTarget Source & Destination"
-    Please note that TrafficTarget is a namespaced resource. Therefore, the source and the destination namespace needs to be explicitly defined.
+!!! Note "TrafficTarget Source & Destination"
+    Please note that TrafficTarget is a namespaced resource.
+    It is important to ensure that the destination namespace is populated, as kubernetes will not autopopulate this field for you.
```

Comment:
```suggestion
    It is important to ensure that the source namespace is populated, as kubernetes will not autopopulate this field for you.
```

Reply:
The entire source field is not mandatory. Only destination is mandatory

Reply:
But a source namespace is mandatory?

For me, It's a bit strange to say that it is `important to ensure that the destination namespace is populated` and in the next sentence that `if the destination namespace is not populated, the TrafficTarget namespace will be used`

Reply:
nope source namespace is not mandatory.

I think the first sentence can be removed.

Reply:
hmm, I'm missing something here. 

So, if the source namespace is not mandatory, is the current code working if the source namespace is not populated? 
I'm thinking that we will have the same problem, right?

Reply:
You don't have to specify a source at all.

No source = All sources allowed.

I'm not sure we handle that correctly at the time being. We should open a new issue for correct source handling in cases of missing source field.

Reply:
👍